src/plugins/tagfilter/tagfilter-fr.hbs

```diff
@@ -785,6 +797,10 @@
 			</div>
 		</div>
 	</div>
+
+	<div class="wb-tagfilter-noresult">
+		<p>Aucun résultat.</p>
+	</div>
```

Comment:
I think this was added by mistake because it is already there at the end of the list of articles.